readinglist/authentication.py

```diff
@@ -1,11 +1,15 @@
+# -*- coding: utf-8 -*-
```

Comment:
To tell Python that the file is in utf-8, it is mostly a best practice nothing that we really need for this specific file.